UserManagement/accounts/migrations/0001_initial.py

```diff
@@ -0,0 +1,107 @@
+# Generated by Django 4.2.3 on 2023-12-06 02:08
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ("auth", "0012_alter_user_first_name_max_length"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="UserProfile",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("password", models.CharField(max_length=128, verbose_name="password")),
+                (
+                    "last_login",
+                    models.DateTimeField(
+                        blank=True, null=True, verbose_name="last login"
+                    ),
+                ),
+                (
+                    "is_superuser",
+                    models.BooleanField(
+                        default=False,
+                        help_text="Designates that this user has all permissions without explicitly assigning them.",
+                        verbose_name="superuser status",
+                    ),
+                ),
+                ("identityId", models.IntegerField(blank=True, null=True)),
+                ("email", models.EmailField(max_length=254, unique=True)),
+                ("full_name", models.CharField(blank=True, max_length=150)),
+                ("date_of_birth", models.DateField(blank=True, null=True)),
+                ("is_active", models.BooleanField(default=True)),
+                ("is_staff", models.BooleanField(default=False)),
```

Comment:
What does it mean is_stuff?

Reply:
we don't have stuff role in our system, I don't want to create inconsistency between naming